code-challenges.test.js

```diff
@@ -1,52 +1,165 @@
+    
+    if (object2(this.number) % 3 === 0) {
+        return `${object2(this.number)} "is divisible by three`}
+    else if (object1(this.number) % 3 !== 0) {
+        return `${object2(this.number)} "is not divisible by three`}
+
+    if (object3(this.number) % 3 === 0) {
+        return `${object3(this.number)} "is divisible by three`}
+    else if (object1(this.number) % 3 !== 0) {
+        return `${object3(this.number)} "is not divisible by three`}
+    
+    }
+    
+    console.log(divideBy3(object1))
+    console.log(divideBy3(object2))
+    console.log(divideBy3(object3))
+
+// //     // Spent WAY WAY to much time on this problem.. Test Failed!
+// //     // cannot get the expected output - Receiving reference errors. Tried different ways to manipulate the code.
+// //     // I thought perhaps the function itself needed to be within the object culy brackets 
+// //     // Shown below:
+
+    const object1 = { number: 15,
+        
+        divideBy3: function() {
+        
+            if ((this.number) % 3 === 0) {
+                `${this.number} "is divisible by three`}
+            else if ((this.number) % 3 !== 0) {
+                `${this.number} "is not divisible by three`}
+        }
+    }
+        Console.log(object1)
+
+// // // This way gave me the same referencing errors
+// // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+
+// // a) Create a test with expect statements for each of the variables provided.
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe("randomNouns1", () => {
+    it("returns an array with all the words capitalized", () => {
+      expect(randomNouns1()).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+
+describe("randomNouns1", () => {
+    it("returns an array with all the words capitalized", () => {
+        expect(randomNouns1()).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+
+// // Good Error
+
 // b) Create the function that makes the test pass.
 
+const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+// Psuedocode
+// Create a function that takes in an array of words and returns an array with all the words capitalized.
+// Input: randomNouns1, randomNouns2
+// Output: Capitalizes the each word in the array set
+//  Use built in methods .toUpperCase and .substring
+
+const capitalLetter = (array) => {
+    return array.map((value) => {
```

Comment:
I don't see anything in your pseudocode about iteration.  Pseudocode is to help make a roadmap on how you intend to solve the problem.  This process should be detailed on each step you plan to take throughout the function.